unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh

```diff
@@ -165,15 +166,16 @@ readFileUsers() {
 
     FILECORRECT=$(grep -Pzc '\A(User:\s*name:\s*\w+\s*password:\s*\w+\s*)+\Z' $FILE)
     if [ $FILECORRECT -ne 1 ]; then
-        logger -e "the password file doesn't have a correct format.
-        It must have this format:
-        User:
-        name: wazuh
-        password: wazuhpasword
-        User:
-        name: kibanaserver
-        password: kibanaserverpassword"
-        exit 1
+	logger -e "The password file doesn't have a correct format.
```

Comment:
Make a variable with formatted text and pass it to the logger.

Reply:
Done in a7004d7